compiler/resolution/functionResolution.cpp

```diff
@@ -2730,9 +2730,9 @@ void printResolutionErrorUnresolved(CallInfo&       info,
 
       } else {
         USR_FATAL_CONT(call,
-                       "type mismatch in assignment from %s to %s",
-                       toString(info.actuals.v[1]->type),
-                       toString(info.actuals.v[0]->type));
+                       "Cannot assign to %s from %s",
+                       toString(info.actuals.v[0]->type),
+                        toString(info.actuals.v[1]->type));
```

Comment:
Github is showing a spacing change on this line. Can you make the spacing match what it was? No tabs.

Reply:
Sure Sir will do it right away.